tests/all/volumes.t

```diff
@@ -238,6 +240,24 @@ puts TEST "kadalu config-snapshot list"
 TEST "kadalu config-snapshot delete snap2 --mode=script"
 puts TEST "kadalu config-snapshot list"
 
+# Tests for renaming of volume
+USE_NODE nodes[0]
+TEST "kadalu volume create DEV/vol20a server1:/exports/vol20a/s1 --no-start"
+TEST "kadalu volume create DEV/vol20b server1:/exports/vol20b/s1 server2:/exports/vol20b/s2 server3:/exports/vol20b/s3 --no-start"
+puts TEST "kadalu pool create DEV2"
+TEST 1, "kadalu volume rename DEV/vol20b DEV2/vol20b"
+
+TEST "kadalu volume delete DEV/vol20a --mode=script"
+TEST 0, "kadalu volume rename DEV/vol20b DEV2/vol20b"
+
+puts TEST "kadalu node list"
+
+# Rename volume back to DEV for successful completion of node delete below
+TEST 0, "kadalu volume rename DEV2/vol20b DEV/vol20b"
+TEST "kadalu volume delete DEV/vol20b --mode=script"
+
+puts TEST "kadalu node list"
+
 nodes.each do |node|
   USE_NODE nodes[0]
   puts TEST "kadalu node remove DEV/#{node} --mode=script"
```

Comment:
The node delete step has `DEV` mentioned as the pool name. After rename some nodes are part of pool `DEV2` so moved all nodes to `DEV`.

Reply:
Good to change `DEV2` here so that it actually tests the delete after rename case as well.

Reply:
changed to DEV2